### PR TITLE
Question about initialization of map grid

### PR DESCRIPTION
**Pull request by jonycfu**
Should I wrap all initialization in one init function?

### CODE REVIEW

**Comment by h8rry** [comment]
Please gitignore `bower_components/`

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+bower_components
```
Comment (h8rry) [inline .gitignore:1]: Please always leave a line break at end of all the files

File: css/custom.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
+html, body, section {
```
Comment (harrychenca) [inline css/custom.css]: Indentations

**Comment by h8rry** [comment]
Indentations needs to be tab_size=2, indent using spaces

**Comment by h8rry** [comment]
The rest looks good. Please let me know once you fix all the indentations

**Pull request merged**